src/StreamJsonRpc.Shared/DataContracts/JsonRpcMessage.cs

```diff
@@ -120,6 +127,13 @@ public object[] GetParameters(ParameterInfo[] parameterInfos, JsonSerializer jso
                 return this.Parameters.ToObject<object[]>();
             }
 
+            if (parameterInfos.Length == 1 && parameterInfos[0].ParameterType == typeof(JToken))
+            {
+                // We want to ignore methods that only have one parameter and the parameter type is JToken.
+                // This is reserved to be called when the parameter is passed as an object.
+                return null;
```

Comment:
That is a breaking change. We shouldn't ignore methods with only one JToken parameters because they can be called passing the parameter value in array. We should be able to call such methods with existing code.